### ISSUE TITLE
Game crash on launch

### ISSUE THREAD

**Issue by puppygirlgamedev** [AeronicaMC/mxTune#5]
My game crashes unexpectedly on start. Im not very good with java crash logs so im posting here with the log
[crash-2022-04-12_02.50.09-client.txt](https://github.com/AeronicaMC/mxTune/files/8468660/crash-2022-04-12_02.50.09-client.txt)
.

**Comment by Aeronica** [comment]
Thanks for the report. The error occurs when mxTune tries to access a private field in the SoundManager class to get the instance of the SoundHandler. I use an access transformer to make if publicly accessible. It's possible this is caused by a rare race condition.

1) Does this happen every time you start? 
2) Does it happen if your remove the CustomNPCs mod?

**Comment by puppygirlgamedev** [comment]
Just tried.
Crashes every time.
Crashes without CustomNPCs.
It is a clean install, no other mods.

**Comment by Aeronica** [comment]
I've created a new version that I will upload to Curse Forge soon. I did not specify the access transformer correctly in the 1.0.2 build. The jar probably worked for myself and others because other mods changed the access modifiers mxTune needs.

Reviewed the debug log and confirmed Forge Mod Loader (FML) found and loaded the accesstransformer.cfg file specified for this new build.

`[15:25:24] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLDeobfTweaker
[15:25:24] [main/DEBUG] [FML]: Loaded 215 rules from AccessTransformer config file forge_at.cfg
[15:25:24] [main/DEBUG] [FML]: Loaded 5 rules from AccessTransformer mod jar file C:\Users\Paul\AppData\Roaming\.mxt12-2\mods\mxtune-1.12.2-1.0.3.jar!META-INF/accesstransformer.cfg
`
